bin/grocery-delivery

```diff
@@ -66,6 +66,12 @@ def read_checkpoint
 end
 
 def full_upload(knife)
+  if GroceryDelivery::Config.berks
+    puts 'berks!'
+    GroceryDelivery::Log.warn('Uploading all cookbooks with berkshelf')
+    GroceryDelivery::Log.debug('using cookbook paths'+GroceryDelivery::Config.cookbook_paths.to_s)
```

Comment:
< 80 chars per line
Prefer string interpolation to concat in ruby, unless breaking lines.
Put spaces around operators.